VCFv4.3.tex

```diff
@@ -326,132 +327,147 @@ \subsubsection{Fixed fields}
+
+If any of the fields is missing, it is replaced with the missing value. For example if the FORMAT is GT:GQ:DP:HQ then $0\mid0:.:23:23,34$ indicates that GQ is missing. Trailing fields can be dropped, with the exception of the GT field, which should always be present if specified in the FORMAT field.
+
+See table~\ref{table:reserved-genotypes} for a list of common, reserved keywords that are standards across the community, and the following points for detailed explanations about the most complex among them. See also Section~\ref{sv-format-keys} for a list of genotype keys reserved for structural variants.
+
+\begin{table}[htbp]
+  \centering
+  \begin{tabularx}{\textwidth}{ | l | l | l | X | }
+      Field		& Number	& Type		& Description \\ \hline
+      AD		& R		& Integer	& Per-sample read depth for each allele \\
+      ADF		& R		& Integer	& Per-sample read depth for each allele on the forward strand \\
+      ADR		& R		& Integer	& Per-sample read depth for each allele on the reverse strand \\
+      DP		& 1		& Integer	& Per-sample read depth \\
+      EC		& A		& Integer	& Expected alternate allele counts for each alternate allele, in the same order as listed in the ALT field (typically used in association analyses) \\
+      FT		& 1		& String	& Filter indicating if this genotype was ``called''. More details in Subsection~\ref{para:genotype-FT} \\
+      GL		& G		& Float		& $log_{10}$-scaled likelihoods for all possible genotypes given the set of alleles defined in the REF and ALT fields. More details in Subsection~\ref{para:genotype-GL} \\
+      GP		& G		& Float		& Genotype posterior probabilities in the range 0 to 1 using the same ordering as the GL field (could be used to store imputed genotype probabilities) \\
+      GQ		& 1		& Integer	& Conditional genotype quality, encoded as a phred quality $-10log_{10}$ p(genotype call is wrong, conditioned on the site's being variant) \\
+      GT   		& 1		& Integer	& Genotype, encoded as allele values separated by either of $/$ or $\mid$. \\
+      HQ		& 2		& Integer	& Haplotype (phred) qualities \\
+      MQ		& 1		& Integer	& RMS mapping quality \\
+      PL		& G		& Integer	& Phred-scaled genotype likelihoods rounded to the closest integer (and otherwise defined precisely as the GL field) \\
+      PQ		& 1		& Integer	& Phasing quality, the phred-scaled probability that alleles are ordered incorrectly in a heterozygote (against all other members in the phase set). More details in Subsection~\ref{para:genotype-PQ} \\
```

Comment:
given that the PQ isn't well defined, I hesitate to set this as number=1 here. while this should be true for diploid samples, one could think of phasing in higher ploidy organisms and there it would be a more complicated object. I think that there should be more freedom in the spec, especially when we are not sure of it's use. (same for PS)